### PR TITLE
Removing console warnings and errors

### LINKED ISSUE #196: Removing Console Warnings
**Describe the bug**
While running the app in the local server several console errors / warnings are shown in the console.

**To Reproduce**
1. Run the app locally using the command `npm run dev`
2. Open the console and check the errors / warnings 

**Expected behavior**
The console must be free from warnings

**Screenshots**
image

**Device Information [optional]:**
- OS: Linux - Debain 
- Browser : Firefox
- Version 22

**Additional context**

**Are you working on this issue? (Yes/No)**
Yes

### PR DESCRIPTION
**Pull request by DhairyaMajmudar**
Fixes #196 

#### Describe the changes you have made in this PR -

 - Upgraded the font awesome from V5 to V6 to remove the console errors / warnings 
 - Commented the hamburger.css file import causing warnings in the console 
 - commented `import { defineProps, defineEmits } from 'vue' ` since its not required any more.  
 - Removed JQuery from `color_theme.scss` 
 - Removed js comments from CSS files 
 - Removed warning from console while building the simulator  

### Screenshots of the changes (If any) -

image

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

**Comment by DhairyaMajmudar** [comment]
@Prerna-0202 @Arnabdaz pls. review the PR and suggest the changes if needed

Thank You : )

**Comment by DhairyaMajmudar** [comment]
@Arnabdaz just ping for PR review

**Comment by Arnabdaz** [comment]
@DhairyaMajmudar can you fix these issues that come up on building the simulator
image

**Review by Arnabdaz — CHANGES_REQUESTED** [review]
@DhairyaMajmudar can you fix these issues that come up on building the simulator
Screenshot from 2023-12-24 16-01-16

**Comment by DhairyaMajmudar** [comment]
Sure @Arnabdaz would be happy to do so : )

**Comment by DhairyaMajmudar** [comment]
Also if this pr looks good can you pls. merge it.

**Comment by Arnabdaz** [comment]
add the changes in this pr itself ... push the changes into the same branch and it would propagate in this pr

**Comment by DhairyaMajmudar** [comment]
Sure @Arnabdaz  I will follow up the same. Also can you pls. brief me where exactly these errors are taking place ?

**Comment by DhairyaMajmudar** [comment]
Sure @Arnabdaz I will follow up the same 😃

**Comment by DhairyaMajmudar** [comment]
@Arnabdaz I have removed the warning from the console while building the simulator for that I followed the approach 

- Analised the files from where warnings were coming.
- Those warning were due to JQuery code present in `color_theme.scss` file 
-  And because of JS comments in `.scss` files.

**Review by Arnabdaz — CHANGES_REQUESTED** [review]
image
there seems to be some issues with the css variable changes take a look carefully and verify

**Comment by DhairyaMajmudar** [comment]
@Arnabdaz This unexpected behavior is because of the white color font `(--text-circuit)` .  But  that was already predefined in the file  [Link to that code](https://github.com/CircuitVerse/cv-frontend-vue/pull/214/files#diff-91ab4cc9b2d20a6a68739cceb166ab5927410f569b69d5958890e832bb93bfbbL17) 

I think previously because of some error the white text color was not applied and by default black font was styled.

**Comment by DhairyaMajmudar** [comment]
This error can be removed by changing `--text-circuit` to `#000` color code.

File: src/styles/color_theme.scss [hunk 1]
```diff
@@ -629,3 +571,145 @@ select {
...
+    background-color: var(--text-panel);
+    color: var(--primary);
+}
+.touchMenuIcon {
+    background-color: var(--primary);
+    border: 2px solid var(--br-secondary);
+    color: var(--text-panel);
+}
+<<<<<<< HEAD
+=======
```
Comment (tachyons) [inline src/styles/color_theme.scss:667]: @DhairyaMajmudar Could you fix the git conflict issues ?

Comment (DhairyaMajmudar) [reply src/styles/color_theme.scss:670]: Yes I am removing conflicts and removing unwanted commits

Comment (DhairyaMajmudar) [reply src/styles/color_theme.scss]: @tachyons I have resolved the merge conflicts and also reverted unwanted changes pls . review pr and merge it.

**Review by Arnabdaz — APPROVED** [review]
LGTM cc @tachyons

File: src/components/MessageBox/messageBox.vue [hunk 2]
```diff
@@ -49,7 +49,7 @@
 
 <script lang="ts" setup>
 import BooleanTable from '@/DialogBox/BooleanTable.vue'
-import { defineProps, defineEmits } from 'vue'
+// import { defineProps, defineEmits } from 'vue'
```
Comment (tachyons) [inline src/components/MessageBox/messageBox.vue:52]: Please remove instead of commenting lines

Comment (DhairyaMajmudar) [reply src/components/MessageBox/messageBox.vue]: Yes @tachyons I have removed extra comments from the pr

**Pull request merged**